resources/config.test.ci.edn

```diff
@@ -28,28 +28,36 @@
-                                   :without-producer {:application-id       "test"
-                                                      :bootstrap-servers    "localhost:9092"
-                                                      :stream-threads-count [1 :int]
-                                                      :origin-topic         "topic"
-                                                      :channels             {:channel-1 {:worker-count [10 :int]
-                                                                                         :retry        {:count   [5 :int]
-                                                                                                        :enabled [true :bool]}}}}}
+            :stream-router        {:default              {:application-id       "test"
+                                                          :bootstrap-servers    "localhost:9092"
+                                                          :stream-threads-count [1 :int]
+                                                          :origin-topic         "topic"
+                                                          :default-key-serde    "org.apache.kafka.common.serialization.Serdes$ByteArraySerde"
+                                                          :default-value-serde  "org.apache.kafka.common.serialization.Serdes$ByteArraySerde"
+                                                          :upgrade-from         "1.1"
+                                                          :channels             {:channel-1 {:worker-count [10 :int]
+                                                                                             :retry        {:count   [5 :int]
+                                                                                                            :enabled [true :bool]}}}
+                                                          :producer             {:bootstrap-servers                     "localhost:9092"
+                                                                                 :acks                                  "all"
+                                                                                 :retries-config                        5
+                                                                                 :max-in-flight-requests-per-connection 5
+                                                                                 :enable-idempotence                    false
+                                                                                 :value-serializer                      "org.apache.kafka.common.serialization.StringSerializer"
+                                                                                 :key-serializer                        "org.apache.kafka.common.serialization.StringSerializer"}}
+                                   :using-string-serde   {:application-id       "test"
```

Comment:
I thought about it. But, adding/deleting existing configs looks messy in the code. Creating a new config simply allows us to just use the keyword everywhere in the tests. 

That way config definition can be separated from its usage. But, I agree that more than two would make for a bad config file.

Additionally, these being reference configs I see the presence of multiple configs useful for a new user who can instantly see how Ziggurat supports multiple streams.

Reply:
how is it messy? There is just one `redef` that you need to do for it.

Again if we agree that more than 2 will not look good, then what happens when there is a need for another topic-entity in the tests? Do we use `redef` there? If we do, won't the tests then become inconsistent?

We can always have good documentation for users to understand the usage, instead of duplicating values in our code.

Reply:
Okay. But, we had two configs (:without-producer) even before this change. At least, this change is consistent with the previous code. So, I would argue that I haven't added a new config here. Just changed the old one.

Reply:
@theanirudhvyas WDYT?